test/e2e/e2e_test.go

```diff
@@ -68,7 +69,7 @@ func TestChannelClusterDefaulter(t *testing.T) {
 	t.Skip("Skipping until https://github.com/knative/eventing-contrib/issues/627 is fixed")
 	cancel := logstream.Start(t)
 	defer cancel()
-	e2ehelpers.ChannelClusterDefaulterTestHelper(t, channelTestRunner, lib.DuplicatePubSubSecret)
+	e2ehelpers.ChannelClusterDefaulterTestHelper(context.TODO(), t, channelTestRunner)
```

Comment:
nit: context.Background()
here and in other places

Reply:
done